modules/fedimint-ln-client/src/lib.rs

```diff
@@ -143,21 +142,12 @@ pub enum InternalPayState {
 pub enum LnPayState {
     Created,
     Canceled,
-    Funded,
-    WaitingForRefund {
-        block_height: u32,
-        gateway_error: GatewayPayError,
-    },
+    Funded { block_height: u32 },
```

Comment:
I think this change also needs a DB migration since old `Funded` states will be 0 bytes in length while the new parsing code will expect 4 bytes.

Reply:
Is this ever stored in the database? It's not `Encodable`, I thought it's just yielded during `subscribe_ln_receive`